edisgo/tools/networkx_helper.py

```diff
@@ -1,11 +1,14 @@
-from networkx import OrderedGraph
+from networkx import Graph
 
 
 def translate_df_to_graph(buses_df, lines_df, transformers_df=None):
-    graph = OrderedGraph()
+    graph = Graph()
 
-    buses = buses_df.index
-    # add nodes
+    buses = []
+    for bus_name, bus in buses_df.iterrows():
```

Comment:
As this is not a perfomance critical part of the code it's not really important, but try to avoid `iterrows` as it is very slow, because it has to create so many `pandas.Series` objects. See [this OS answers](https://stackoverflow.com/a/55557758/12460232) for better ways. E.g. this should work:
```
buses = [
    (bus_name, {"pos": (x, y)})
    for bus_name, x, y in buses_df[["x", "y"]].itertuples()
]
```